hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/BootstrapExecutor.java

```diff
@@ -191,11 +194,32 @@ private void initializeTable() throws IOException {
         .fromProperties(props)
         .setTableType(cfg.tableType)
         .setTableName(cfg.targetTableName)
+        .setRecordKeyFields(props.getString(KeyGeneratorOptions.RECORDKEY_FIELD_NAME.key()))
+        .setPartitionFields(props.getString(KeyGeneratorOptions.PARTITIONPATH_FIELD_NAME.key(), null))
+        .setPreCombineField(props.getString(HoodieWriteConfig.PRECOMBINE_FIELD_NAME.key()))
```

Comment:
Probably, we need to set this in the test as well. I see some CI test failure due to precombine config not set in the test.

Reply:
For the precombine config, the logic now falls back to the default value if not provided by the user.